docs/content/preview/yugabyte-platform/manage-deployments/manage-runtime-config.md

```diff
@@ -0,0 +1,49 @@
+---
+title: Manage Runtime configuration
+headerTitle: Manage Runtime configuration
+linkTitle: Manage Runtime configuration
+description: Manage your universes by setting runtime configuration keys based on different scopes.
+menu:
+  preview_yugabyte-platform:
+    identifier: manage-runtime-config
+    parent: manage-deployments
+    weight: 100
+type: docs
+---
+
+YugabyteDB Anywhere has a configuration mechanism that is fully compatible with the existing [typesafe config](https://github.com/lightbend/config) format and has the following additional properties:
+
+- Runtime/Dynamic: The configuration keys can be set or changed at runtime via the API.
+- Persistent: The site specific configuration changes won't be lost in an install.
+- Backwards compatibility: Fully compatible with existing mechanism with default values residing in the file `application.conf` or `reference.conf`.
+- Scoped: Configuration can be applied to a smaller scope, for example, even a single universe can be configured to have certain features.
+- Hierarchical: Scopes are nested and allow for overriding outer scope values in inner scope.
+
+Currently, the scopes can be categorized as follows:
+
+| Scope | Description | User Access |
+|:--- |:--- | :--- |
+| Global | Applies to the entire platform, persists across installs and any value defined in this scope overrides the defaults. | Super Admin |
```

Comment:
Is this only for YB support/personnel? If so why is it in customer docs?

Reply:
Its both for YB Support team as well super admins in the customer organization who can use this feature to leverage and modify the runtime settings. I hope that answers your question.

Reply:
@ddhodge in YBA a super admin is a user role. Not our support person.
But many of these are advanced knobs that we have to tweak during debugging.